components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/src/test/java/org/wso2/carbon/identity/cors/mgt/core/test/CORSManagementServiceTests.java

```diff
@@ -86,6 +95,8 @@ public class CORSManagementServiceTests {
     private MockedStatic<IdentityTenantUtil> identityTenantUtil;
     private MockedStatic<ApplicationManagementService> applicationManagementService;
     private MockedStatic<IdentityDatabaseUtil> identityDatabaseUtil;
+    private OrganizationManager organizationManager;
+    private OrgResourceResolverService orgResourceResolverService;
```

Comment:
Let's add a unit test to verify cors origin merging behavior as well

Reply:
It might be too complex to implement this behaviour IMO